contrib/mesos/pkg/executor/executor.go

```diff
@@ -329,6 +356,43 @@ func (k *KubernetesExecutor) LaunchTask(driver bindings.ExecutorDriver, taskInfo
 	go k.launchTask(driver, taskId, pod)
 }
 
+func (k *KubernetesExecutor) handleChangedApiserverPod(pod *api.Pod) {
+	taskId := pod.Annotations[meta.TaskIdKey]
+	if taskId == "" {
+		log.V(5).Infof("ignoring pod update for %s/%s because %s annotation is missing", pod.Namespace, pod.Name, meta.TaskIdKey)
```

Comment:
doesn't this seem like a bug/error condition that we'd want to report on with a warning level?